lib/taskana-core/src/test/java/acceptance/taskrouting/TaskRoutingAccTest.java

```diff
@@ -0,0 +1,53 @@
+package acceptance.taskrouting;
+
+import static org.junit.Assert.assertEquals;
```

Comment:
please use junit 5 assertions. junit 4 is no longer available with my PR

Reply:
Done